apps/ui-library/content/docs/nextjs/social-auth.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: Social Authentication
+description: Social authentication block for Next.js
+---
+
+<BlockPreview name="social-auth/auth/login" />
+<Callout>
+  The block is using Github provider by default, but can be easily switched by changing a single
+  parameter.
+</Callout>
+
+## Installation
+
+<BlockItem name="social-auth-nextjs" description="All needed components for the social auth flow" />
+
+## Folder structure
+
+This block includes the [Supabase client](/ui/docs/nextjs/client). When installing, you can skip overwriting it.
+
+<RegistryBlock itemName="social-auth-nextjs" />
+
+## Usage
+
+Once you install the block in your Next.js project, you'll get all the necessary pages and components to set up a social authentication flow.
+
+### Getting started
+
+First, add a `.env` file to your project with the following environment variables:
+
+```env
+NEXT_PUBLIC_SUPABASE_URL=
+NEXT_PUBLIC_SUPABASE_ANON_KEY=
+```
+
+- If you're using supabase.com, you can find these values in the [Connect modal](https://supabase.com/dashboard/project/_?showConnect=true) under App Frameworks or in your project's [API settings](https://supabase.com/dashboard/project/_/settings/api).
+
+- If you're using a local instance of Supabase, you can find these values by running `supabase start` or `supabase status` (if you already have it running).
+
+### Setting up third party providers
+
+We support a wide variety of social providers that you can use to integrate with your application. The full list is available [here](https://supabase.com/docs/guides/auth/social-login).
+The block is using PKCE flow and Github as a provider, but you could easily change it by changing the `provider` field in `supabase.auth.signInWithOAuth` call.
+
+### Setting up routes and redirect URLs
+
+1. Set the site URL in the [URL Configuration](https://supabase.com/dashboard/project/_/auth/url-configuration) settings in the Supabase Dashboard.
+1. Set up the Next.js route that users will visit to reset or update their password. Go to the [URL Configuration](https://supabase.com/dashboard/project/_/auth/url-configuration) settings and add the `forgot-password` route to the list of Redirect URLs. It should look something like: `http://example.com/auth/forgot-password`.
+
+1. Update the redirect paths in the `login-form.tsx` components to point to the logged-in routes in your app.
```

Comment:
Update the redirect paths in `login-form.tsx` to point to your app’s logged-in routes. Our examples use `/protected`, but you can set this to whatever fits your app.

?

Reply:
Fixed in all auth blocks.